### ISSUE TITLE
Use `basename` to simplify routes

### ISSUE THREAD

**Issue by caasi** [SenseTW/sensetw#104]
**Is your feature request related to a problem? Please describe.**
It's possible to set the base URL in react-router v4:

```
<BrowserRouter basename="/calendar"/>
<Link to="/today"/> // renders <a href="/calendar/today">
```

**Comment by pm5** [comment]
Closed by 28ae9cc2af262fa5e944aada10b4ead039abdba2

**Comment by caasi** [comment]
如果未來需要用 history 來移動到別的 route ，還是會遇上得自己接 `process.env.PUBLIC_URL` ，先記在這裡。
